app/src/main/java/wei/mark/standouttest/floatingfolders/FloatingFolder.java

```diff
@@ -1,4 +1,4 @@
-package wei.mark.floatingfolders;
+package wei.mark.standouttest.floatingfolders;
```

Comment:
I see you've moved floatingfolders/ into app/.

Restore the old folder hierarchy where example/ and floatingfolders/ are both top level modules. You will have to add an additional build gradle config for floatingfolders/